external_resources_io/input.py

```diff
@@ -41,3 +41,45 @@ def read_input_from_env_var(var: str = "INPUT") -> dict[str, Any]:
     b64data = os.environ[var]
     str_input = base64.b64decode(b64data.encode("utf-8")).decode("utf-8")
     return json.loads(str_input)
+
+
+def get_ai_provision_data() -> AppInterfaceProvision:
+    """Get the AppInterfaceProvision from the input data file."""
+    ai_input = read_input_from_file(
+        file_path=os.environ.get("ER_INPUT_FILE", "/inputs/input.json"),
```

Comment:
Please move the `os.environ.get("ER_INPUT_FILE", "/inputs/input.json")` to the `read_input_from_file` method to avoid the duplicated `/inputs/input.json` definition.